### ISSUE TITLE
Problem with MAE 

### ISSUE THREAD

**Issue by taekyungss** [taekyungss/dreamdiffusion#10]
- [x] MAE validation loss 추가 및 Overfitting 여부 확인
- [x] VAE 일때 loss 나오게 코드 수정
- [x] early stop 코드 추가해서 lr , mask_ratio parameter 조정해보기
- [x] data 구성해서 latent space -> T-SNE 시각화 해보고 분석 (person, class)
- [x] MAE말고 다른 방법론은 없을까? Reading & Thinking
